tests/e2e-instrumentation/instrumentation-nodejs-multicontainer/00-install-instrumentation.yaml

```diff
@@ -7,23 +7,20 @@ spec:
     - name: OTEL_TRACES_EXPORTER
       value: otlp
     - name: OTEL_EXPORTER_OTLP_ENDPOINT
-      value: http://localhost:4317
+      value: http://localhost:4318
```

Comment:
Very likely we don't need this since we are setting `.spec.exporter.endpoint`, right?

Reply:
I left it intentionally, once this telemetry data assertion for all the languages is done. I'll work on asserting the instrumentation CR and its parameter overrides as well.